composer/trainer/trainer.py

```diff
@@ -1604,7 +1590,8 @@ def _train_loop(self) -> None:
 
                 if int(self.state.timestamp.batch_in_epoch) == 0:
                     self.engine.run_event(Event.EPOCH_START)
-                    self.logger.data_epoch({'epoch': int(self.state.timestamp.epoch)})
+                    self.logger.log_metrics({'trainer/epoch': int(self.state.timestamp.epoch)},
```

Comment:
I think we explicitly wanted `epoch` to not be indented with `trainer` previously (cc.. @abhi-mosaic ? )

Reply:
ok fixed